### PR TITLE
Fixes #15222: Add a generic method to ensure that a kernel module is not loaded

### PR DESCRIPTION
**Pull request by Fdall**
https://issues.rudder.io/issues/15222

### CODE REVIEW

**Comment by Fdall** [comment]
PR updated with a new commit

File: tree/30_generic_methods/kernel_module_not_loaded.cf [hunk 1]
```diff
@@ -0,0 +1,53 @@
+# @name Kernel module not loaded
```
Comment (ncharles) [inline tree/30_generic_methods/kernel_module_not_loaded.cf]: Missing license header

File: tree/30_generic_methods/kernel_module_not_loaded.cf [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+# @description Ensure that a given kernel module is not loaded on the system
+# @documentation Ensure that a given kernel module is not loaded on the system.
+#   If a module loaded is found, it will try to unload it via modprobe.
```
Comment (ncharles) [inline tree/30_generic_methods/kernel_module_not_loaded.cf]: "If the module is loaded, it will try to unload it using modprobe"

**Comment by Fdall** [comment]
PR updated with a new commit

**Comment by ncharles** [comment]
:+1:

**Comment by Fdall** [comment]
OK, squash merging this PR

**Comment by Fdall** [comment]
OK, merging this PR

**Comment by Fdall** [comment]
OK, merging this PR

**Pull request merged**